File: src/main/java/moe/zhi/model/DateTime.java
package moe.zhi.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

/**
 * v1.0 fix me
 *
 * @author zhimoe
 * @version 1.0
 * @since 2022/7/3
 */
@Data
public class DateTime {
    @JsonProperty("$id")
    String id;
    String currentDateTime;
    String currentFileTime;
    String ordinalDate;
    String timeZoneName;
}

File: src/main/java/moe/zhi/model/DateTime.kt
package moe.zhi.model

import com.fasterxml.jackson.annotation.JsonProperty

/**
 * v1.0 fix me
 *
 * @author zhimoe
 * @version 1.0
 * @since 2022/7/3
 */
open class DateTime {
    @JsonProperty("\$id")
    var id: String? = null
    var currentDateTime: String? = null
    var currentFileTime: String? = null
    var ordinalDate: String? = null
    var timeZoneName: String? = null
}